authorize_request_handler_oidc_request_test.go

```diff
@@ -213,7 +213,7 @@ func TestAuthorizeRequestParametersFromOpenIDConnectRequest(t *testing.T) {
 				},
 			}
 
-			err := f.authorizeRequestParametersFromOpenIDConnectRequest(context.Background(), req)
+			err := f.authorizeRequestParametersFromOpenIDConnectRequest(context.Background(), req, false)
```

Comment:
Would we need a test suite which has this set to true?

Reply:
The `true` case is covered in the PAR request. The final parameter is `isPARRequest`.